src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateGlobalUsings.targets

```diff
@@ -50,8 +50,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <ItemGroup>
       <Compile Include="$(GeneratedGlobalUsingsFile)" />
-      <!-- Performing a clean and waiting without a build results in build warnings in VS due to all the missing namespaces, so we'll avoid recording it in FileWrites when building in VS -->
-      <FileWrites Include="$(GeneratedGlobalUsingsFile)" Condition="'$(BuildingInsideVisualStudio)' != 'true'" />
+      <!-- Performing a clean and waiting without a build results in build warnings in VS due to all the missing namespaces, so we'll avoid recording it in FileWrites -->
```

Comment:
Nit: I'd mention that we plan to solve this another way and put the `FileWrites` item back in a future release.  I'd probably also leave `<FileWrites Include="$(GeneratedGlobalUsingsFile)" />` commented out for reference.

Reply:
I filed a tracking issue - https://github.com/dotnet/sdk/issues/21667 - to undo these PRs once we have the work in VS done. I'd prefer that to leaving in comments in code / commented out code.